ldj-maven-plugin/pom.xml

```diff
@@ -48,6 +48,7 @@
     <maven.compiler.target>17</maven.compiler.target>
     <maven.compiler.source>17</maven.compiler.source>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <sonar.skip>true</sonar.skip>
```

Comment:
For this one, we should probably include it (even if that means the coverage % is then lower):

* If there are vulnerabilities/etc in the Maven plugin, we probably want to know about those.
* There's not really a reason why the Maven plugin doesn't or shouldn't need tests, they just don't exist yet.

For the other ones, I agree entirely that skipping Sonar is sensible.

Reply:
i see your point.